src/lib/containers/entity/SchemaDrivenAnnotationEditor/template/ArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  ADDITIONAL_PROPERTY_FLAG,
+  ArrayFieldTemplateItemType,
+} from '@rjsf/utils'
+import React, { useEffect, useState } from 'react'
+import { FormGroup } from 'react-bootstrap'
+import { getTemplate, getUiOptions, ArrayFieldTemplateProps } from '@rjsf/utils'
+import { Collapse } from '@material-ui/core'
+import { HelpOutline } from '@material-ui/icons'
+
+/** The `ArrayFieldTemplate` component is the template used to render all items in an array.
+ *
+ * @param props - The `ArrayFieldTemplateItemType` props for the component
+ */
+function ArrayFieldTemplate<T = any, F = any>(
```

Comment:
Moved from `CustomArrayFieldTemplate`. Since it now uses other templates that didn't exist before, it was substantially rewritten.